### DISCUSSION TITLE
Can't click on nav items

### DISCUSSION THREAD

**Discussion by iTheMaverick** [community/community#70752] [Programming Help] [answered]
Hi all, I'm new web designer and am still learning a few things but know enough to be able to make websites. I've encountered an issue with one that I'm working on, after changing a few things in CSS my nav items can no longer be clicked and :hover is no longer functioning and I cannot figure out what the problem is coming from

![image](https://github.com/community/community/assets/141296807/5af585cc-4824-425c-bcc4-bd0a846a9d8c)
![image](https://github.com/community/community/assets/141296807/4b1b7bd7-306d-4eff-b7d3-b21da7bef7b3)
![image](https://github.com/community/community/assets/141296807/a6ed8a2a-2ace-4f7f-8646-2f0a8669de5b)
![image](https://github.com/community/community/assets/141296807/93a03416-bba3-490c-88fa-94fb809acd5e)
![image](https://github.com/community/community/assets/141296807/45b70e98-8930-4dd8-8277-ce6aadc09b7c)
![image](https://github.com/community/community/assets/141296807/2b7fa26d-eba4-48c6-8a3b-68f2f63866dc)
![image](https://github.com/community/community/assets/141296807/d6dc5276-1969-4542-8f16-49fb2dd3cf15)

(I tried pasting the code directly but it kept embedding)

Any help is appreciated, thanks.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by deep100x** [comment]
yoy should provide code instaed of picture

**Reply by iTheMaverick** [reply]
`<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />

    <link rel="stylesheet" href="CSS/contactus.css" />
    />

    <title>Toplist Perfumes</title>
  </head>
  <body>
    <header>
      <h1>Contact Us</h1>
      <hr class="Topbar" />
      <navbar class="Topnav">
          <a class="Home" href="index.html">Home</a>
          <a class="NewArrivals" href="newarrivals.html">New Arrivals</a>
            <a class="BrandCollection" href="brandcollection.html">Brand Collection</a>
              <a class="Catalog" href="catalog.html">Catalog</a>
                <a class="ContactUs" href="contactus.html">Contact Us</a>
                  <a class="FinanceYourPurchase" href="financeyourpurchase.html">Finance Your Purchase</a>
<a class="Login" href="login.html">Login</a>
<a class="Search" href="search.html">Search</a>
<a class="Cart" href="cart.html">Cart</a>
        <img
        class="HeaderLogo"
        src="img/TopListScentsWebsite.png"
        alt="ToplistLogo"
        />
      </navbar>
    </header>
    <div>
      <h2 class="About">About Toplist Perfumes</h2>
      <p class="Info">
        835 N Michigan Ave, Chicago, IL, 60611 Unit 6005<br><br
        >Phone: +1 847-497-8459 <br>Email: Toplistscents@gmail.com</br><br>Monday-Friday 11am-7pm<br>Sunday 11am–6pm
      </p>
      <hr>
      <h3>Careers</h3>
      <p class="Careers">For careers, contact Toplistscents@gmail.com</p>
      <hr>
      <h4>Inquiries</h4>
      <p class="Inquiries">Contact Toplistscents@gmail.com</p>
    </div>
    <footer>
      <hr class="Footer">
    </footer>
  </body>
</html>`

`* {
  padding: 0;
  margin: 0;
  box-sizing: 0;
}

body {
  font-family: sans-serif;
  background-color: #f3f3f3;
}

.Topnav {
  margin: auto;
  padding-top: 2%;
  position: fixed;
  text-align: center;
  font-size: 1.8vh;
  white-space: nowrap;
}

.HeaderLogo {
  position: fixed;
  padding: auto;
  margin: auto;
  width: 20%;
  left: 3%;
  top: 0.5%;
}

.Topbar {
  position: fixed;
  width: 100%;
  height: 13%;
  right: 0%;
  background: #f3f3f3;
  top: 0%;
  padding: auto;
  margin: auto;
  border: none;
  border-bottom: 5px solid #1a1a1a;
}

.Home {
  margin: auto;
  color: #1a1a1a;
  text-decoration: none;
  margin-right: 2%;
  margin-left: 70%;
}

.Home:hover {
  border-bottom: 1px solid #1a1a1a;
}

.NewArrivals {
  margin: auto;
  color: #1a1a1a;
  text-decoration: none;
  margin-right: 2%;
}

.BrandCollection {
  margin: auto;
  color: #1a1a1a;
  text-decoration: none;
  margin-right: 2%;
}

.BrandCollection:hover {
  border-bottom: 1px solid #1a1a1a;
}

.Catalog {
  margin: auto;
  color: #1a1a1a;
  text-decoration: none;
  margin-right: 2%;
}

.Catalog:hover {
  border-bottom: 1px solid #1a1a1a;
}

.ContactUs {
  color: #1a1a1a;
  text-decoration: none;
  border-bottom: 1px solid #1a1a1a;
  margin-right: 2%;
}

.ContactUs:hover {
  border-bottom: 1px solid #1a1a1a;
}

.FinanceYourPurchase {
  color: #1a1a1a;
  text-decoration: none;
  margin-right: 2%;
}

.FinanceYourPurchase:hover {
  border-bottom: 1px solid #1a1a1a;
}

.Login {
  color: #1a1a1a;
  text-decoration: none;
  margin-left: 40%;
}

.Login:hover {
  border-bottom: 1px solid #1a1a1a;
}

.Search {
  color: #1a1a1a;
  text-decoration: none;
  margin-left: 2%;
}

.Search:hover {
  border-bottom: 1px solid #1a1a1a;
}

.Cart {
  color: #1a1a1a;
  text-decoration: none;
  margin-left: 2%;
}

.Cart:hover {
  border-bottom: 1px solid #1a1a1a;
}

h1 {
  position: absolute;
  padding-top: 8%;
  padding-left: 45%;
  margin: auto;
  font-size: 4vh;
  text-align: center;
  font-family: system-ui;
  font-weight: 350;
}

h2 {
  position: relative;
  padding-top: 11%;
  padding-left: 25%;
  margin: auto;
  font-size: 3vh;
  text-align: left;
  font-family: system-ui;
  font-weight: 350;
}

.Info {
  position: relative;
  padding-top: 1%;
  padding-left: 25%;
  margin: auto;
  font-size: 2vh;
  font-family: system-ui;
  font-weight: 100;
}

.Careers {
  position: relative;
  padding-top: 1%;
  padding-left: 25%;
  margin: auto;
  font-size: 2vh;
  font-family: system-ui;
  font-weight: 100;
}

.Inquiries {
  position: relative;
  padding-top: 1%;
  padding-left: 25%;
  padding-bottom: 5%;
  margin: auto;
  font-size: 2vh;
  font-family: system-ui;
  font-weight: 100;
}

h3 {
  position: relative;
  padding-top: 1%;
  padding-left: 25%;
  margin: auto;
  font-size: 3vh;
  text-align: left;
  font-family: system-ui;
  font-weight: 350;
}

h4 {
  position: relative;
  padding-top: 1%;
  padding-left: 25%;
  margin: auto;
  font-size: 3vh;
  text-align: left;
  font-family: system-ui;
  font-weight: 350;
}

hr {
  margin-left: 15%;
  margin-right: 15%;
  margin-top: 2%;
}

.Footer {
  position: relative;
  width: 100%;
  height: 12%;
  background: #1a1a1a;
  padding: auto;
  margin: auto;
}
`

**Comment by deep100x** [comment, marked as answer]
h2 {
position: relative;
padding-top: 11%;
padding-left: 25%;
margin: auto;
font-size: 3vh;
text-align: left;
font-family: system-ui;
font-weight: 350;
}

This is messing up Navigation bar !!
If you remove or comment padding in h2{...} your navugation bar will start
working !!

Also you given font-weight 350 which is wrong !
font-weight : 100 or 200 or 300 or 400 or 500 or 600 or 700 or 800 or 900 ;

Kindly contact me if you face any issue with your website
